src/ck-core/cklocation.h

```diff
@@ -444,20 +494,22 @@ class CkLocMgr : public IrrGroup
   {
     return CMK_RANK_0(map->procNum(mapHandle, idx));
   }
-  inline bool isHome(const CkArrayIndex& idx) const
+  inline bool isHome(const CkArrayIndex& idx) const { return homePe(idx) == CkMyPe(); }
```

Comment:
These probably don't need `inline` unless something weird is going on with includes.